stix_shifter_utils/stix_translation/src/json_to_stix/json_to_stix_translator.py

```diff
@@ -13,6 +13,28 @@
 FIRST_OBSERVED_KEY = 'first_observed'
 LAST_OBSERVED_KEY = 'last_observed'
 
+STIX_2_1_PROPERTY_UPDATE_MAP = {
```

Comment:
this mapping dictionary makes me think. should we have to_stix_map for 2.1 as well?